src/main/java/Commands.java

```diff
@@ -0,0 +1,22 @@
+public enum Commands {
+    LIST("list"),
+    TODO("todo"),
+    DEADLINE("deadline"),
+    EVENT("event"),
+    EXIT("bye"),
+    MARK("mark"),
+    UNMARK("unmark"),
+    INVALID_COMMAND("invalid command!");
+    public final String string;
```

Comment:
Good way to define all the commands, to make editing your code in the future easy and more recognizable!